src/components/EventEditView/EventEditView.tsx

```diff
@@ -0,0 +1,205 @@
+import { useEffect, useState } from 'react'
+import { createStyles, makeStyles, Grid, TextField } from '@material-ui/core'
+import { useHistory, useParams } from 'react-router-dom'
+import { Box } from '@mui/material'
+
+import EventTypes from '../EventTypes/EventTypes'
+import SelectWithLoading from '../DropDown/SelectWithLoading'
+import TextFieldWithValidation from '../TextField/TextFieldWithValidation'
+import FormButtons from '../FormButtons/FormButtons'
+import EventsApi from '../../api/events'
+import { EventEditViewProps } from '../../shared/entities/props/eventEditViewProps'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    disabled: {
+      pointerEvents: 'none'
+    },
+    container: {
+      flexFlow: 'column',
+      textAlign: 'center',
+    },
+    textField: {
+      width: '100%',
+    },
+    wideInput: {},
+  })
+)
+
+export default function EventEditView({
+  headquarters,
+  tags,
+  validation,
+  isLoading,
+}: EventEditViewProps): JSX.Element {
+
+  const [getData, setData] = useState<{[key: string]: string}>({})
+  const classes = useStyles()
+  const history = useHistory()
+  const { id } = useParams<string>()
+
+  const fetchEventById = async (eventId: string | undefined) => {
+    const api = new EventsApi()
+    try {
+      const response = await api.getById(eventId)
```

Comment:
destructuring please
```suggestion
      const {  name, ...others } = await api.getById(eventId)
```

Reply:
Done